tests/backends/test_numba_object.py

```diff
@@ -1368,11 +1368,11 @@ def get_true(v):
 def test_operator_truth():
     @numba.njit
     def get_true(v):
-        return True if v else False
+        return True if v else False  # noqa: SIM210
 
     @numba.njit
     def get_false(v):
-        return False if v else True
+        return False if v else True  # noqa: SIM211
```

Comment:
We don't support `not v`, which is a little strange, but just noqa'd it. `not bool(v)` probably would have worked, since I think `bool(v)` above worked.